packages/btcindexer/wrangler.jsonc

```diff
@@ -45,6 +45,7 @@
     "SUI_FALLBACK_ADDRESS": "0xf82fd2198d5af45bfc2a7f9a5df4fb30f3c3abf4a8a71b0e4bd415eabda99ff5",
     "CONFIRMATION_DEPTH": "4",
     "MAX_RETRIES": "1",
+    "SCAN_BATCH_SIZE": "100",
```

Comment:
The name is not clear enough, too generic. Let's rename to `BTC_BLOCK_PROCESSING_BATCH`. 
Also, rename `MAX_RETRIES`  to `MAX_NBTC_MINT_TX_RETRIES`